include/evmc/evmc.h

```diff
@@ -324,7 +324,7 @@ struct evmc_result
     /**
      * The reference to output data.
      *
-     *  The output contains data coming from RETURN opcode (iff evmc_result::code
+     *  The output contains data coming from RETURN opcode (if evmc_result::code
```

Comment:
This is not a typo, it means "if and only if".

Reply:
I wasn't sure if you actually `iff` or just a typo. I can add it to a whitelist.